source/tasks/CreateOctopusRelease/CreateOctopusReleaseV3/task.json

```diff
@@ -60,12 +60,15 @@
             "helpMarkDown": "Version 3 of this task has limited support for spaces. We recommend using version 4 of this task for a better experience."
         },
         {
-            "name": "ProjectFilter",
-            "type": "string",
-            "label": "Filter project by name",
+            "name": "ProjectGroup",
```

Comment:
We now display project group dropdowns that then let you select projects within that group.